metrics/register.go

```diff
@@ -0,0 +1,38 @@
+package metrics
+
+import (
+	"fmt"
+
+	"go.opencensus.io/stats/view"
+)
+
+var registeredViews = map[string][]*view.View{}
+
+func RegisterViews(name string, views []*view.View) error {
+	views, ok := registeredViews[name]
+	if ok {
+		return fmt.Errorf("duplicate registration of views by name %s", name)
+	}
+
+	if err := view.Register(views...); err != nil {
+		return err
+	}
+
+	registeredViews[name] = views
+	return nil
+}
+
+func LookupViews(name string) ([]*view.View, error) {
+	views, ok := registeredViews[name]
+	if !ok {
+		return nil, fmt.Errorf("no view %s found registered", name)
+	}
+	return views, nil
+}
+
+func AllViews() (views []*view.View) {
+	for _, v := range registeredViews {
+		views = append(views, v...)
+	}
+	return
```

Comment:
Let's make the return explicit.

```suggestion
	return views
```